### PR TITLE
Complete the first phase of the project

### PR DESCRIPTION
**Pull request by Hombre2014**
In the first phase of the project the following steps have been completed:
- Setup webpack
- Create index.html with empty To-Do List placeholders elements
- Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object contains three keys:
-- description [string].
-- completed [boolean].
-- index: [number].
- Write a function to iterate over the tasks array and populate an HTML element for each task.
- On page load render the dynamically created tasks in the dedicated placeholder. The list appears in order of the index values for each task.
- Create a style.css and set rules for the To-Do List. CSS must be loaded by Webpack Style/CSS Loader. The list should be a clone of the part of the minimalist project captured in the video shown in the README.md file.
- All source files (index.html, index.js, and style.css) are located in /src directory and distribution files are generated by webpack and served by the webpack dev server from /dist folder.

### CODE REVIEW

**Comment by D4nielJ** [comment]
Hi @Hombre2014, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not include a short summary of changes introduced in the pull request.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Hombre2014** [comment]
> Hi @Hombre2014,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that does not include a short summary of changes introduced in the pull request. Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Hi @D4nielJ ! Is this a short summary you are talking about, is the description section of the pull request, that I did not fill? Please, reply. Thanks for your time and consideration.

**Comment by D4nielJ** [comment]
> Hi @D4nielJ ! Is this a short summary you are talking about, is the description section of the pull request, that I did not fill? Please, reply. Thanks for your time and consideration.

I'm sorry I did not see your reply on time.

Yes, the short summary must be in the description of the Pull Request. You have included it correctly now.

File: README.md [hunk 1]
```diff
@@ -32,35 +32,33 @@ In this project, you will build a simple HTML list of To Do tasks. The list will
...
-N/A
+- N/A
 
 ### Usage
 
+- N/A
 
 ### Run tests
 
+- N/A
```
Comment (dasileker) [inline README.md]: - [ ] Please add how you can download the repo and run it locally.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Hombre2014,** 

### **To highlights 👇✔️**

- The **PR** is good.
- The commits are good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Yuriy, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**